waku/message_test.go

```diff
@@ -182,9 +184,6 @@ func TestMessageSeal(t *testing.T) {
 	params.TTL = 1
 
 	env := NewEnvelope(params.TTL, params.Topic, msg, time.Now())
-	if err != nil {
-		t.Fatalf("failed Wrap with seed %d: %s.", seed, err)
-	}
```

Comment:
Just so I'm trying to sneak this through, I want to point out the I've removed this because the condition is always false because err is always nil. The err is tested on line 181 and doesn't change after that check and before the check I've removed.